confgenerator/testdata/goldens/logging-receiver_elasticsearch/golden/linux-gpu/fluent_bit_main.conf

```diff
@@ -47,22 +47,6 @@
     Tag               elasticsearch.elasticsearch_gc
```

Comment:
Note: I believe in this particular receiver there shouldn't be any changes in `fluent_bit_main.conf`. Hopefully addressing the updates in `elasticsearch.go` should remove this changes.